okio/src/main/java/okio/Base64.java

```diff
@@ -114,27 +114,42 @@ private Base64() {
       '5', '6', '7', '8', '9', '+', '/'
   };
 
+  private static final byte[] URL_MAP = new byte[] {
```

Comment:
Obviously multiple ways to achieve this effect. I chose the 2am, simple, and tired approach.